src/http/upstream.rs

```diff
@@ -0,0 +1,15 @@
+/// Define a static upstream peer initializer
+///
+/// Initializes the upstream 'get', 'free', and 'session' callbacks and gives the module writer an
+/// opportunity to set custom data.
+/// Load Balancing: <https://nginx.org/en/docs/dev/development_guide.html#http_load_balancing>
+#[macro_export]
+macro_rules! http_upstream_peer_init {
```

Comment:
For these functions, the upstream modules all use this format: `ngx_http_upstream_init_[MODULE_NAME]_peer`

I kind of followed the typedef for the signature, but accidentally transposed peer and init. We can name it identical to that typedef (minus the ngx, this repo tends to omit that prefix).

I also placed it in upstream.rs because the typedef is in ngx_http_upstream.h, so it's a clue and alignment to the original.

If we were more faithful to the NGINX name it would be `http_upstream_init_peer_pt`. I think keeping this in upstream is a helpful alignment to NGINX code base.

Reply:
Renamed to reflect the function type it defines.